textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/neextractor/NEExtractorTestConstants.java

```diff
@@ -1,83 +1,81 @@
 package edu.uci.ics.textdb.dataflow.neextractor;
 
-import edu.uci.ics.textdb.api.common.*;
-import edu.uci.ics.textdb.common.constants.SchemaConstants;
-import edu.uci.ics.textdb.common.field.*;
-import edu.uci.ics.textdb.common.utils.Utils;
-import edu.uci.ics.textdb.dataflow.neextrator.NamedEntityExtractor;
-
 import java.text.ParseException;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
+import edu.uci.ics.textdb.api.common.Attribute;
```

Comment:
I strongly suggest you undo all the changes (mainly reformatting) in the `neextractor` module.  It's not the scope of this PR, and may generate conflicts.  Please treat it as a general comment.